java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -20,39 +20,20 @@ private NativeUtils() {
     }
 
     /**
-     * This will read from /proc/version to attempt to find the Linux distribution.
-     * @return The Linux distribution or null if the version cannot be found.
-     * @throws IOException If an I/O error occurs
-     */
-    public static String getDistroName() throws IOException {
-        Pattern distroRegex = Pattern.compile("[^(]+\\([^(]+\\([^(]+\\(([A-Za-z\\s]+).*");
-        BufferedReader reader = new BufferedReader(new FileReader("/proc/version"));
-        String distro;
-        try {
-            Matcher line = distroRegex.matcher(reader.readLine());
-            distro = line.matches() ? line.group(1) : null;
-        }
-        finally {
-            reader.close();
-        }
-        return distro;
-    }
-
-    /**
-     * This will attempt to find the Linux version by making use of {@code lsb_release -r}
-     * @return The Linux version or null if the version cannot be determined.
-     * @throws IOException If an I/O error occurs
+     * Shell call to lsb_release to get OS info
+     * @param lsb_args parameters to lsb_release.  EX: "-i" or "-a"
+     * @param regexp a regular expression pattern used to parse the response from lsb_release
```

Comment:
JavaDoc will complain here as well because you need an @throws and @return.  Sorry for being a stickler here, but once we move to Java 8 these JavaDoc warnings become compiler errors.